tests/tests_pytorch/loggers/test_wandb.py

```diff
@@ -497,3 +499,44 @@ def test_wandb_logger_download_artifact(wandb, tmpdir):
     WandbLogger.download_artifact("test_artifact", str(tmpdir), "model", True)
 
     wandb.Api().artifact.assert_called_once_with("test_artifact", type="model")
+
+
+@mock.patch("lightning.pytorch.loggers.wandb._WANDB_AVAILABLE", True)
+@mock.patch("lightning.pytorch.loggers.wandb.wandb", mock.Mock())
+@pytest.mark.parametrize(("log_model", "expected"), [("True", True), ("False", False), ("all", "all")])
+def test_wandb_logger_cli_integration(log_model, expected, monkeypatch, tmp_path):
+    """Test that the WandbLogger can be used with the LightningCLI.
+
+    Regression test for
+    https://github.com/Lightning-AI/lightning/issues/18370
+
+    """
```

Comment:
This context is not necessary, there's git blame for it

```suggestion
    """Test that the WandbLogger can be used with the LightningCLI."""
```